_posts/2025-10-16-guests.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Calling all Guest Posters!
+tags: GitHub MathTech Blogging
+author: Steven Clontz
+comments: true
+---
+
+Continuing my series on [Getting Started with GitHub](/2025/10/10/github-history.html), I thought I should
+point out that you can directly contribute your own guest post to our blog using your GitHub account,
+and it's probably easier than you think!
+
+The first step is to create a [fork](https://g4m.code4math.org/ch-collaboration.html#def-forks) of
+[our `mathtechorg.github.io` repository](https://github.com/StevenClontz/mathtechorg.github.io).
+With just two clicks, you'll have your own personal copy of our entire blog on your own GitHub account!
+Assuming you're already logged into GitHub (or [created your free account](https://github.com/signup)),
+click "Fork", and then on the next page click the green "Create Fork" button with all the default settings.
```

Comment:
```suggestion
The first step is to log into GitHub (or [create your free account](https://github.com/signup)).
Then with just two clicks, you can create a
[fork](https://g4m.code4math.org/ch-collaboration.html#def-forks) of
[our `mathtechorg.github.io` repository](https://github.com/StevenClontz/mathtechorg.github.io).
and have your own personal copy of our entire blog on your own GitHub account!
Just click "Fork", and then on the next page click the green "Create Fork" button with all the default settings.
```

Reply:
@Tanaquil18 what do you think? (You can wordsmith directly by clicking the +- button above the comment field if you like.)